onnxruntime/test/onnx/main.cc

```diff
@@ -335,35 +335,22 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
   // clang-format on
 
 #ifdef _WIN32
```

Comment:
_WIN32 Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64. Otherwise, undefined.
_WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined.

ref: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2017